ComfyUI/custom_nodes/luma_photon_node/luma_photon_node.py

```
@@ -0,0 +1,225 @@
+from __future__ import annotations
+import torch
+import numpy as np
+from PIL import Image
+import os
+import io
```

Ruff (0.12.2)

`io` imported but unused

Remove unused import: `io`

(F401)